### ISSUE TITLE
[TextFields] `helperText` should have an accessibility analogue

### ISSUE THREAD

**Issue by romoore** [material-components/material-components-ios#2841]
### Overview
The `helperText` property on `MDCTextInputController` has no accessibility equivalent. This makes it difficult for clients to set different values for the displayed value and what VoiceOver reads.

### Expected behavior

As an example, formatting a number may be handled differently for displayed and VoiceOver values:

**Displayed text**
"+82-2-312-3456 or 02-312-3456"

**VoiceOver Label**
"The domestic or international format for the telephone number."

### Reproduction steps
 - Detailed steps to reproduce

### Version number
44.3.0

**Comment by willlarche** [comment]
perhaps additional data like 
`-(void)setHelperText:(NSString*)helperText accessibilityThing:(NSString*)accessibilityThing`
